src/api/utils/pagination.ts

```diff
@@ -1,6 +1,8 @@
 import debug from 'debug';
+import { Options, OptionsOfJSONResponseBody } from 'got';
 import got from 'got';
```

Comment:
The default import `got` is no longer used in this file, so we can remove this import.